### PR TITLE
Bump version in Configuration sample

### PR DESCRIPTION
**Pull request by ytsarev**
### Description of your changes

Bump version in Configuration schema.

The assumption is that we will have/start with the current tag increment inherited from https://github.com/upbound/platform-ref-aws/blob/main/examples/configuration.yaml

I have:

- [x] Read and followed Upbound's [contribution process](https://git.io/fj2m9).
- [ ] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR, as appropriate.

### How has this code been tested

uptest below

### CODE REVIEW

File: examples/configuration.yaml [hunk 1]
```diff
@@ -3,4 +3,4 @@ kind: Configuration
 metadata:
   name: cofiguration-aws-network
 spec:
-  package: xpkg.upbound.io/upbound/platform-aws-network:0.1.0
+  package: xpkg.upbound.io/upbound/platform-aws-network:0.7.0
```
Comment (haarchri) [inline examples/configuration.yaml:6]: v0.7.0 ?

Comment (ytsarev) [reply examples/configuration.yaml:6]: @haarchri 
In the description of this PR 

>The assumption is that we will have/start with the current tag increment inherited from >https://github.com/upbound/platform-ref-aws/blob/main/examples/configuration.yaml

Up to discussion. We can obviously start to release everything with a totally independent version tagging schema

**Pull request closed without merging**